app/questionnaire/placeholder_parser.py

```diff
@@ -8,13 +8,16 @@ class PlaceholderParser:
     final values
     """
 
-    def __init__(self, language, answer_store=None, metadata=None):
+    def __init__(self, language, answer_store=None, metadata=None, schema=None):
         self._answer_store = answer_store or AnswerStore()
         self._metadata = metadata
+        self._schema = schema
         self._transformer = PlaceholderTransforms(language)
 
     def _lookup_answer(self, answer_id):
-        answer = self._answer_store.get_answer(answer_id)
+        answer = self._answer_store.get_answer(
```

Comment:
Seems like the problem of getting a default answer is one that's going to regularly come up. Can we add a method to answer_store to return it given a schema object?

Reply:
Answer store shouldn't have knowledge of default answers. That should stay with the QuestionnaireSchema